sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job_limits.py

```diff
@@ -83,6 +75,16 @@ class SweepJobLimits(JobLimits):
     :type timeout: int
     :param trial_timeout: Sweep Job Trial timeout value in seconds.
     :type trial_timeout: int
+
+    .. admonition:: Example:
```

Comment:
Is admonition some sort of docstring keyword? Edit - nevermind, found out below.